v1/heroku.json

```
@@ -1 +1 @@
-{"owners":[{"id":1589,"type":"github_user"},{"id": 177877, "type": "github_user"},{"id": 6271256, "type": "github_user"}, {"id": 59744, "type": "github_user"}]}
+{"owners":[{"id":1589,"type":"github_user"},{"id": 177877, "type": "github_user"}, {"id": 59744, "type": "github_user"}, {"id": 917672, "type": "github_user"}, {"id": 88346006, "type": "github_user"}, {"id": 501702, "type": "github_user"}]}
```

```suggestion
{"owners":[{"id":1589,"type":"github_user"},{"id": 177877, "type": "github_user"},{"id": 59744, "type": "github_user"}, {"id": 917672, "type": "github_user"},{"id": 88346006, "type": "github_user"},{"id": 501702, "type": "github_user"}]}
```

@joshwlewis actually can you cut the spaces?